src/Blaster.WebApi/Features/CapabilityDashboard/index.cshtml

```diff
@@ -94,7 +94,7 @@
             </div>
        
         <!--- Summary section --->
-            <div class="container box">
+            <div class="container box" style="display:none">

```

Comment:
Might be good to mention in either PR title or description that this is being hidden. A quick look at the title currently(Clean ui no awsperms) would make one think that only that(AWS Permissions view) is being affected.

Nothing too critical though, certainly not a blocker. It is mentioned in the individual commit where the change takes place. So assuming this commit doesn't get squashed, it should be fine.